python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```diff
@@ -691,6 +691,14 @@ $defs:
                       type: int
                       convert_types:
                         - str
+                    ipv4_acl_in:
```

Comment:
what about SVIs as requested in the initial issues?

Reply:
we will address SVI in another PR

Reply:
https://github.com/aristanetworks/avd/blob/0b1272b9c4513aa99bdfbd986f5d5a69005fb267/python-avd/pyavd/_eos_designs/structured_config/network_services/ip_access_lists.py#L109

I think this already preset.

Reply:
yes ! I just realized this. Thanks for commenting .. It was implemented in this PR: https://github.com/aristanetworks/avd/pull/4096